.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
+]
```

Comment:
Haven't used Elixir in a bit, can you give me a tl;dr on what this is? Not blocking, just curious.